components/SpotifyInterface/SpotifyInterface.c

```diff
@@ -117,7 +105,16 @@ static void Spotify_MainTask(void *pvparameters)
             {
                 if (Spotify_HttpServerServiceInit() == true)                                                        // Init the webserver in initialization
                 {
-                    PrivateHandler.Status = LOGIN;                                                                   // if webserver and mDNS run successfully, go to LOGIN
+                    if (SpiffsExistenceCheck(InterfaceHandler->ConfigAddressInSpiffs) == true)                      // Check if the refresh token exists in the spiffs
```

Comment:
this function made static, and SpiffsIsExists function use for external usage.

Reply:
I see, I will replace this funciton with the new thread-safe function in the updated spiffs component

Reply:
as we discussed, I am going to merge this PR and then migrate to the updated SPIFFS module in another PR